### DISCUSSION TITLE
Option to have a non-coloured icon of Viva Engage

### DISCUSSION THREAD

**Discussion by michaelmaillot** [microsoft/fluentui#32811] [Ideas]
Hello,

To complete [#25038](https://github.com/microsoft/fluentui/discussions/25038), it's strange to have a coloured version of Viva Engage icon, especially as it's the only one [here](https://developer.microsoft.com/en-us/fluentui#/styles/web/icons):

![image](https://github.com/user-attachments/assets/d3b4398b-d134-4825-98de-cf9c67448f03)

Furthermore, as it has replaced Yammer, icon should be updated to be available in Fluent UI font, SVG,... In non-coloured version.

**Comment by tudorpopams** [comment]
Hello, we have an internal channel where you can request a new icon to be added to the set. Feel free to reach out to me directly so I can point you to it.

**Reply by michaelmaillot** [reply]
Hi @tudorpopams, if it's more relevant for requesting a new icon, I'm ok to be into this!

**Reply by michaelmaillot** [reply]
A friendly reminder 🙂

**Reply by michaelmaillot** [reply]
Hello @tudorpopams,

Any news regarding this?
